Apps/FCM/ios/FCMSampleApp/AppDelegate.swift

```diff
@@ -0,0 +1,131 @@
+import UIKit
+import React
+import CioMessagingPushFCM
+import CioTracking
+import Firebase
+import FirebaseMessaging
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
```

Comment:
Is there a reason why you converted to Swift ? Large number of our customers (almost all) use Objective C and not swift. I have experienced customers asking for Objective C samples in specific because Swift apps confuse them. The reason I believe is that by default, react native templates generate ObjC code.

Reply:
Mainly because:

- We didn't have a swift example in new react native sample apps while previous sample app had [AppDelegate in swift](https://github.com/customerio/amiapp-reactnative/blob/main/ios/AppDelegate.swift)
- Having more familiarity with swift made it easier for me to fix any issues occurred while adding/modifying the code

I think we have seen customers using swift for react native which probably is because converting initial Objective-C template to swift isn't very difficult and make it easier to add further changes to native code. But I do agree that more customers are still on Objective-C because react native uses it in default template.

Keeping in mind that our docs covers all variations of Objective-C/Swift, it still might be a good idea to have one version with AppDelegate in Objective-C and another in Swift. However, I'll wait for more feedback on this, if the squad's preference is to convert it, I'll revert it back to Objective-C.

Reply:
I have also see the majority of customers use Obj-C. In fact, I am not sure if I have experienced a customer using Swift? 

I agree that having 1 swift and 1 obj-c app could be valuable, however, I am leaning towards both being obj-c or the APN app being the 1 with Swift. Why? Because FCM integration requires AppDelegate changes in it. More changes then an APN app does. Therefore, I think the FCM sample app being Obj-c is going to be more valuable to the majority of our customers. 

This is my current recommendation, but I am open to other ideas. I share this only for consideration.

Reply:
Alright, we have a clear winner 😁 I'll revert this back to Objective-C and update the PR

Reply:
I would also prefer ObjC over Swift in this case ! Apologies Rehan for the additional work !

Reply:
No worries Aman, that was good suggestion and helpful for most customers.

Reply:
I tried converting code in a single commit so it is easier to review. The [conversion is complete](https://github.com/customerio/customerio-reactnative/pull/182/commits/c868771223658c7b352329275f5a692465335204), but deep links (both app scheme and universal scheme) only works if app is killed. I'm not yet sure what's breaking this as it was working with Swift version. If anyone has suggestions, please let me know. Would appreciate some 👀 on rest of the code as well.

Reply:
Thanks for the quick debug Aman, [deep links are fixed](https://github.com/customerio/customerio-reactnative/pull/182/commits/79cd8eb54305bd243cc787d2de01496604c4d550) now 🎉